finder-app/writer.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+writefile="$1"
+writestr="$2"
+
+# Check if writefile is specified
+if [ -z "$writefile" ]; then
+    echo "Error: writefile not specified."
+    exit 1
+fi
+
+# Check if writestr is specified
+if [ -z "$writestr" ]; then
+    echo "Error: writestr not specified."
+    exit 1
+fi
+
+#print statements if any of the parametersvwere not specified
+#Reference manual : https://man7.org/linux/man-pages/man3/stapvars.3stap.html
+if [ $# -ne 2 ]; then
```

Comment:
@Rishikesh38 I think you should re-order to put this at the front - as is I think your script will exit early due to the errors above without printing usage when you don't specify arguments.